recipe/meta.yaml

```diff
@@ -12,9 +12,9 @@ source:
     - nolalsuite-in-pyproject.patch
 
 build:
-  skip: true  # [py<39 or win]
+  skip: true  # [win]
```

Comment:
just out of curiosity: why this change? I thought that the only way to prevent building for python3.8 was to add this (commit 161d21042b0b432678cf6a1c7b3713f7e297fcef)